network-store-model/src/main/java/com/powsybl/network/store/model/ReactiveCapabilityCurveHolder.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.network.store.model;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.powsybl.iidm.network.ReactiveLimitsKind;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Charly Boutier <charly.boutier at rte-france.com>
+ */
+public interface ReactiveCapabilityCurveHolder {
+
+    ReactiveLimitsAttributes getReactiveLimits();
```

Comment:
can ReactiveLimitsAttributes be changed to ReactiveCapabilityCurvePointAttributes ?

Reply:
No, because we have to also use MinMax here. Instead, renamed this interface to ReactiveLimitHolder.